File: c++/Processor.cpp
class Processor{
    private:
        //atribut
        string name;
        int price;

    public:
        //konstruktor
        Processor(){
        }
        
        Processor(string name, int price){
            this->name = name;
            this->price = price;
        }

        //setter
        void setName(string name){
            this->name = name;
        }

        void setPrice(int price){
            this->price = price;
        }

        //getter
        string getName(){
            return this->name;
        }

        int getPrice(){
            return this->price;
        }

        void printProcessor(){
            cout << "Processor~ " << endl;
            cout << "Name  : "<< this->name << endl;
            cout << "Price : Rp "<< this->price << endl;
        }

        //destruktor
        ~Processor(){
        }

};

File: java/Processor.java
class Processor{

    //atribut
    private String name;
    private int price;

    Processor(){
    }

    //setter
    public void setName(String name){
        this.name = name;
    }

    public void setPrice(int price){
        this.price = price;
    }

    //getter
    public String getName(){
        return this.name;
    }

    public int getPrice(){
        return this.price;
    }

    //prosedur untuk keluaran
    void printProcessor(){
        System.out.println ("Processor~ ");
        System.out.println ("Name       : " + this.name);
        System.out.println ("Price      : Rp " + this.price);
        }
}